nutrition_labels/human_accuracy.py

```diff
@@ -35,8 +40,8 @@
 
     # find what got labelled as what
     conf_matrix = pd.DataFrame(confusion_matrix(both_labelled['Nonie code'], both_labelled['Liz code']),
-                               columns=list(range(1, 7)),
-                               index=list(range(1, 7)))
+                               columns=list(range(1, 5)),
+                               index=list(range(1, 5)))
     print('confusion matrix of Liz labels and Nonie lables')
```

Comment:
If this is the range of the confusion matrix it misses out 5 and 6 labeled.

Reply:
its because I converted 4 and 6 to 5s before this, but actually its true that this matrix is useful to show before that cleaning, so I will move it around

Reply:
ohh yeah that makes sense